scripts/ci/check-pr

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+exec |& tee -a "$ARTIFACTS/check-pr.txt"
```

Comment:
$ARTIFACTS doesn't seem to be set, You should set a default.

Reply:
Try `/tmp/artifacts`, if you look at CI logs you're just getting a relative dir I think

Reply:
Is that dir captured by prow?

Reply:
I'm pretty sure that's what artifact dir is normally set to